packages/buttons/stories/9-ToggleIcon.stories.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { ToggleIconButton } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import Icon from '@zendeskgarden/svg-icons/src/16/eye-stroke.svg';
+
+export default {
+  title: 'Components/Buttons/ToggleIconButton',
+  component: ToggleIconButton
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isPrimary,
+  isBasic,
+  isPill,
+  focusInset,
+  isRotated,
+  isPressed
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <ToggleIconButton
+          aria-label="See more"
+          isDanger={isDanger}
+          size={size}
+          isPrimary={isPrimary}
+          isBasic={isBasic}
+          isPill={isPill}
+          focusInset={focusInset}
+          isRotated={isRotated}
+          isPressed={isPressed}
+        >
+          <Icon />
+        </ToggleIconButton>
+      </Col>
+    </Row>
+  </Grid>
+);
+
+Default.argTypes = {
```

Comment:
If possible, recommend using the same general ordering as seen under the "Toggle Button" section of https://zendeskgarden.github.io/react-components/buttons/#basic. There is a logic to the order of the knobs.

Reply:
I was able to sort the ordering for most examples by moving the control definition ordering and changing how the `propTypes` ordering. However, this particular Story did not get sorted even when control definitions and `PropTypes` were moved around.

Reply:
hmm, ok. gtk.